theano/gof/cmodule.py

```diff
@@ -377,7 +377,7 @@ def is_same_entry(entry_1, entry_2):
 
 def get_module_hash(src_code, key):
     """
-    Return an MD5 hash that uniquely identifies a module.
+    Return a SHA256 (for FIPS compatibility) hash that uniquely identifies a module.
```

Comment:
I don't think that we care about FIPS which is a security standard for encryption in the US government.

I would avoid referencing it.